.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>taplib-adql21</name>
```

Comment:
This should not have been 2.1 without full 2.1 support merged in, applies to later changes also.